packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -48,11 +48,14 @@ export interface SelectProps<T extends SelectOption = SelectOption> {
   wrapperCSS?: CSSObject;
 }
 
-const ArrowDown = (theme: Theme) => `
-    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-        <path fill="${theme.general.color}" fill-rule="evenodd" clip-rule="evenodd" d="M7.99963 12.5711L15.6565 4.91421L14.2423 3.5L7.99963 9.74264L1.75699 3.5L0.342773 4.91421L7.99963 12.5711Z"/>
-    </svg>
-`;
+const ArrowDown = (
+  <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+    <path
+      fill="${black}"
```

Comment:
This looks a little suspicious, I don't see any `black` variable in the scope and a variable named `black` seems weird :thinking:

Reply:
Good catch, this is here just for testing, needed to be removed